mapserver/c2cgeoportal.map.mako

```diff
@@ -76,6 +76,7 @@ MAP
     END
 
     LEGEND
+        KEYSIZE 20 20
```

Comment:
The result I have...
![size](https://cloud.githubusercontent.com/assets/353872/15357939/1e9a449c-1d02-11e6-9b45-afc8c4655dd3.png)